packages/patternfly-4/react-core/src/components/BackgroundImage/BackgroundImage.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly-next/components/BackgroundImage/styles.css';
+
+export const propTypes = {
+  /** Content rendered inside the background */
+  children: PropTypes.node,
+  /** Additional classes added to the background. */
+  className: PropTypes.string
+};
+
+export const defaultProps = {
+  children: '',
+  className: ''
+};
+
+const BackgroundImage = ({ className, children, ...props }) => (
+  <div {...props} className={css(styles.backgroundImage, className)}>
+    {children}
```

Comment:
isn't the backgroundImage fixed? so probably wouldn't matter if children were nested or siblings